traffic_ops/testing/api/v4/server_server_capabilities_test.go

```diff
@@ -216,25 +189,26 @@ func TestServerServerCapabilities(t *testing.T) {
 						})
 					case "POST":
 						t.Run(name, func(t *testing.T) {
-							alerts, reqInf, err := testCase.ClientSession.CreateServerServerCapability(ssc, testCase.RequestOpts)
-							for _, check := range testCase.Expectations {
-								check(t, reqInf, nil, alerts, err)
-							}
-						})
-					case "PUT":
```

Comment:
Why take out PUT?

Reply:
server_server_capabilities doesn't have a PUT method, just moved that test for that route into its own test file: multiple_server_capabilities_test.go